data/announcements/example.yaml

```diff
@@ -0,0 +1,28 @@
+---
+### Example announcement
+#
+# Leave this example announcement in place. It serves as a template
+# and as documentation by example.
+#
+# 🛈  Announcement changes require approval from @kubernetes/steering-committee
+#
+#
+
+# priority is required; largest number wins tie break
+priority: 0
+dateConstraints:
+  # dateConstraints.startTime is optional
+  startTime: 2020-01-01T00:00:00
+  # dateConstraints.endTime is required
+  endTime: 2020-04-01T00:00:00
+# style is optional; if using, set both color and background
+# choose a dark color for the background
+style: >-
+  color: #fff; background: #000;
+# title is optional
+title: Sample announcement
+# message is required
+message: |-
+  Marklar marklar marklar marklar marklar.  
+  Marklar marklar.
+  [Marklar](https://en.wikipedia.org/wiki/Starvin%27_Marvin_in_Space#Plot)!
```

Comment:
if we wanted to keep this translatable, we could go with titleKey and messageKey, use things like

```
titleKey: announcement_blm_title
messageKey: announcement_blm_message
```

and have

```
[announcement_example_title]
other = "Example announcement title."

[announcement_example_message]
other = "Marklar marklar marklar marklar marklar.<br />Marklar marklar.<br />[Marklar](https://en.wikipedia.org/wiki/Starvin%27_Marvin_in_Space#Plot)!"
```

Reply:
I don't feel strongly about the announcement translation, so if that is important to the docs team, I'm happy to defer. The scheduling and approver visibility were the biggest requests

Reply:
Yes, this example should work if you only show one announcement at a time?
[announcement.en]
title = "Kubecon"
message = "Kubecon now"
name = "kubecon"

Reply:
> Yes, this example should work if you only show one announcement at a time?

We want a unique key per announcement, so lagging translations don't display unexpected content

Reply:
My recommendation is _not_ to localize announcements; at least, not yet. It's difficult to ensure that localization teams can update messages promptly. I'm also not sure what the approval would need to look like.

Let's postpone looking into those details for a separate issue and the PR(s) that implement it.